core/src/test/java/org/elasticsearch/search/aggregations/pipeline/InternalSimpleValueTests.java

```diff
@@ -30,19 +31,18 @@
 public class InternalSimpleValueTests extends InternalAggregationTestCase<InternalSimpleValue>{
 
     @Override
-    protected InternalSimpleValue createTestInstance(String name,
-            List<PipelineAggregator> pipelineAggregators, Map<String, Object> metaData) {
-        DocValueFormat formatter = randomFrom(DocValueFormat.BOOLEAN, DocValueFormat.GEOHASH,
-                DocValueFormat.IP, DocValueFormat.RAW);
-        double value = randomDoubleBetween(0, 100000, true);
+    protected InternalSimpleValue createTestInstance(String name, List<PipelineAggregator> pipelineAggregators,
+            Map<String, Object> metaData) {
+        DocValueFormat formatter = randomFrom(DocValueFormat.BOOLEAN, DocValueFormat.RAW, new DocValueFormat.Decimal("###.##"));
```

Comment:
I added a `randomNumericDocValueFormat()` method in core which seems appropriate for this.